.github/workflows/haskell-ci.yml

```diff
@@ -26,14 +26,18 @@ jobs:
     strategy:
       matrix:
         include:
-          - ghc: 8.10.3
+          - ghc: 9.0.1
             allow-failure: false
           - ghc: 8.8.4
             allow-failure: false
-          - ghc: 8.4.3
+          - ghc: 8.4.4
```

Comment:
It looks like we now list 8.4.4 twice but no 8.10 version. That doesn't seem quite right.

I would prefer we keep testing 8.10.

Reply:
It seems I read 8.4.4 as 8.8.4. But the point about 8.10 remains.

Reply:
I added 8.10.3 again to `tested-with` and to the CI.